From 35228aecb3fde5a898cf556e619e94b1e71392b1 Mon Sep 17 00:00:00 2001
From: Mikolaj Izdebski <mizdebsk@redhat.com>
Date: Mon, 20 May 2013 07:33:07 +0200
Subject: [PATCH] Set default editor to vi

Upstream uses nano as editor, probably because it default in Debian.
The default text editor in Fedora is vi.
---
 awesomerc.lua.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -up awesome-3.5.3/awesomerc.lua.in.use-vi-instead-of-nano awesome-3.5.3/awesomerc.lua.in
--- awesome-3.5.3/awesomerc.lua.in.use-vi-instead-of-nano	2014-03-29 18:17:20.000000000 +0100
+++ awesome-3.5.3/awesomerc.lua.in	2014-04-02 18:12:13.652025715 +0200
@@ -42,7 +42,7 @@ beautiful.init("@AWESOME_THEMES_PATH@/de
 
 -- This is used later as the default terminal and editor to run.
 terminal = "xterm"
-editor = os.getenv("EDITOR") or "nano"
+editor = os.getenv("EDITOR") or "vi"
 editor_cmd = terminal .. " -e " .. editor
 
 -- Default modkey.
